test/vm.install

```diff
@@ -27,10 +27,17 @@ if type firewall-cmd >/dev/null 2>&1; then
 fi
 
 # grab a few images to play with; tests run offline, so they cannot download images
-podman rmi --all
-podman pull quay.io/libpod/busybox
-podman pull quay.io/libpod/alpine
-podman pull quay.io/cockpit/registry:2
+if grep -q ID.*fedora /etc/os-release; then
+    version=$(. /etc/os-release && echo "$VERSION_ID")
```

Comment:
Please use /usr/lib/os-release, and you can source this unconditionally. Just check `[ "$ID" = "fedora" ]`